book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+
+
+
+OSH projects emerge as **a response to a concrete need**. 
+For example, you may need to process lots of samples, or to measure a parameter you were not considering before, or to take measurements outside of the lab. 
+With the analyzing of your need, your OSH project starts.
+It is advisable to scope the project at this point (see project scope chapter, In preparation).
+It may indeed be useful to draw a roadmap, look and find future users, contributors and manufacturers (who may have complementary skills to yours), and decide for a license.
+It is reasonable to invest quite a lot of time in looking for similar projects.
+OSH are often difficult to find and comprehend, but finding projects may save you a lot of time and hassle.
+Indeed, you may learn a lot about your needs, refine your ideas, and may even find collaborators while browsing existing OSH projects.
+
+
+
+You may then look and test some ideas  to address your particular need, a process called **prototyping**.
+After many iterations and testing, you will be able to select a prototype for further development.
+The design that solves your need but is not yet complete nor stable is called a **demonstrator**.  
```

Comment:
what does "stable" mean in the hardware context? could be good to clarify that, e.g. "ready for production" or "passes sufficient quality assurance tests"

Reply:
The idea of a stable design comes from integrated product design and means that defined acceptance criteria are met at the end of each development phase before the next phase is commenced. Once a product is tested with defect rates within acceptable limits, a design may also be complete. It may then be qualified and replicated within a controlled and capable process. OSH development however tends to be more iterative and incremental. In highly collaborative settings, sharing early and often can be key. So, a stable design may simply refer to a design that is ready to be released once a ceartain stage in the development has been reached.